app/src/main/java/org/simple/clinic/sync/indicator/SyncIndicatorViewController.kt

```diff
@@ -52,11 +54,11 @@ class SyncIndicatorViewController @Inject constructor(
     val syncProgress = lastSyncedStateStream.filter { it.lastSyncProgress != null }
 
     val showSyncIndicatorState = Observables
-        .combineLatest(screenCreated, syncProgress)
-        { _, stateStream ->
+        .combineLatest(screenCreated, syncProgress, configProvider.toObservable())
```

Comment:
Can I propose renaming `SyncProgress` to `SyncResult` or something? It doesn't feel like we're capturing the progress. What do you think?

Reply:
We are capturing the sync progress and not the result. 
We already renamed this from `syncResult` -> `syncProgress`
Refer https://github.com/simpledotorg/simple-android/pull/485#discussion_r256263171

Reply:
The initial PR did have it named `SyncResult`. I didn't feel that was right because it doesn't actually record the result of a sync, but whether a sync was `syncing`, `done` or `failed`, and I suggested `SyncProgress` as a name for it. I would've suggested `SyncStatus`, but since we already use that for our DB models, I felt it would not be a good choice.

Reply:
Heh okay